app/controllers/cookies_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class CookiesController < ApplicationController
+  skip_before_action :authenticate_user!
+  skip_authorization_check
+
+  def new
+    usage_cookie = cookies[:analytics_cookies_set]
+    @cookie = Cookie.new(analytics: usage_cookie)
+    store_previous_page_url
+  end
+
+  def create
+    @cookie = Cookie.new(cookie_params[:cookie])
+    if @cookie.valid?
+      set_cookies
+      set_flash_notification
+      redirect_to cookies_settings_path
+    else
+      render :new
+    end
+  end
+
+  def cookie_details; end
```

Comment:
I assume your code analyzer hasn't complained about this empty def.

I would suggest you get a second opinion around the best practices here.

Reply:
I'll leave this in so it is consistent with [this](https://github.com/ministryofjustice/laa-court-data-ui/blob/61a6f6e9b799f418792cc4f6a771972b69aac5ec/app/controllers/help_controller.rb#L7)